### PR TITLE
Misc/minor annoyances

### PR DESCRIPTION
**Pull request by fightingtexasaggie**
js sort tested on chrome, Firefox, and Edge. Since we're going to have a big testing session with lots of people in as DRPMs, I thought it nice to finally alphabatize the DIF title list.

Also, the link on the default menu now points to the generic change password logic that does a change if logged in, and a reset if not logged in...this mirrors what the drop down under the 'help' menu does.

### CODE REVIEW

File: templates/Default/index.html.twig [hunk 1]
```diff
@@ -5,7 +5,7 @@
     <h3>Links to:</h3>
     <ul>
         <li><a href="{{  path("pelagos_app_ui_account_default") }}">Request Account</a></li>
-        <li><a href="{{  path("pelagos_app_ui_account_passwordreset") }}">Password Reset</a></li>
+        <li><a href="{{  path("pelagos_app_ui_account_password") }}">Change/Reset password</a></li>
```
Comment (praneethpr) [inline templates/Default/index.html.twig]: Text change is fine, but the route name change is not necessary and it needs to be changed in the Controllers as well, if we want to make this change

File: public/js/dif.js [hunk 2]
```diff
@@ -287,8 +287,19 @@ $(document).ready(function()
             }
         });
     }
+
+    var select = $('select#researchGroup');
+        select.html(select.find('option').sort(function(x, y) {
```
Comment (praneethpr) [inline public/js/dif.js]: 👍

File: templates/Default/index.html.twig (hunk 1 above)
Comment (praneethpr) [reply templates/Default/index.html.twig]: Never mind, I thought this route dint exist

**Comment by fightingtexasaggie** [comment]
It's only an annoying to DRPMs, and it's an unholy waste of everyone's time to deal with long unsorted lists, but I'll ping the PO.

**Pull request closed without merging**